xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -70,9 +77,105 @@ ConfigUpdate build() {
    * The results include configurations for a single upstream cluster, such as endpoint discovery
    * type, load balancing policy, connection timeout and etc.
    */
-  // TODO(zdapeng): content TBD.
   static final class ClusterUpdate {
+    private final String clusterName;
+    private final String edsServiceName;
+    private final String lbPolicy;
+    private final boolean enableLrs;
+    private final String lrsServerName;
+
+    private ClusterUpdate(String clusterName, String edsServiceName, String lbPolicy,
+        boolean enableLrs, @Nullable String lrsServerName) {
+      this.clusterName = clusterName;
+      this.edsServiceName = edsServiceName;
+      this.lbPolicy = lbPolicy;
+      this.enableLrs = enableLrs;
+      this.lrsServerName = lrsServerName;
+    }
+
+    String getClusterName() {
+      return clusterName;
+    }
+
+    /**
+     * Returns the resource name for EDS requests.
+     */
+    String getEdsServiceName() {
+      return edsServiceName;
+    }
+
+    /**
+     * Returns the policy of balancing loads to endpoints. Always returns "round_robin".
```

Comment:
We don't need `Always returns "round_robin"` in javadoc because in the future we will support WRR.

Reply:
At current stage, leaving such a note sounds better as it gives important information. We should remove this after we support policies other than round robin.

Reply:
The API is not doing what the javadoc says. Options:
- Move `Always returns "round_robin"` from javadoc to `//` comments.
- Return hard coded "round_robin" so that API does what javadoc says. (Bad)
- Change javadoc `Always returns "round_robin"` to `Only "round_robin" is supported as of now`.

Reply:
Done. The third option sounds best to me.